tests/tests_unit/rules/test_models/test_information_rules.py

```diff
@@ -28,6 +24,121 @@ def david_spreadsheet() -> dict[str, dict[str, Any]]:
     }
 
 
+def correct_solution_rules():
```

Comment:
just a personal preference
```suggestion
def valid_solution_rules():
```